features/logging/logging_acceptance.feature

```diff
@@ -3,8 +3,6 @@ Feature: Logging smoke test case
   # @author gkarager@redhat.com
   # @case_id OCP-37508
   @admin
-  @destructive
```

Comment:
@anpingli The `@destructive` tag can't be removed because the scenario calls https://github.com/openshift/verification-tests/blob/master/features/logging/logging_acceptance.feature#L11, and this step must have `@destructive` label: https://github.com/openshift/verification-tests/blob/master/features/step_definitions/logging.rb#L199

Reply:
@QiaolingTang  What does ensure_destructive_tagged do? Could we delete it in logging.rb#L199?

Reply:
@anpingli same as `@destructive`:  https://github.com/openshift/verification-tests/blob/master/doc/admin_features.adoc#tags
I added this method to some steps because deploy/remove operators/clusterlogging can affect all the logging cases. From my side, removing this method won't affect anything as all the logging scenarios are tagged with `@destructive`. 
@pruan-rht  any thoughts about removing `ensure_destructive_tagged` from https://github.com/openshift/verification-tests/blob/master/features/step_definitions/logging.rb#L199 and https://github.com/openshift/verification-tests/blob/master/features/step_definitions/logging.rb#L31?

Reply:
It's good to have `ensure_destructive_tagged` in the step definitions if the step is destructive to your other tests

Reply:
I think that is OK to remove this tag. First:  the step is only called by this acceptance case.  Second:  We have destructive tag for all senearios expect this one. Third: we are running caess in sequnce in Logging Pipeline. so that won't make trouble to Logging testing.